src/inc/eventtracebase.h

```diff
@@ -606,9 +607,22 @@ namespace ETW
 
         }MethodStructs;
 
+        enum class JitOptimizationTier
+        {
+            Unknown, // to identify older runtimes that would send this value
+            MinOptJitted,
+            QuickJitted,
+            Optimized,
```

Comment:
You should split out Tier1 from optimized as they will (eventually) diverge.

Reply:
Just `Optimized` and `Tier1Optimized`? Or do you prefer something like `OptimizedForBlended` and `OptimizedForSpeed`, etc. where tier 1 would map to the speed one, and untiered cases would map to the blended one?

I had kept `Optimized` generic for now, as in the future if we have info that says pregenerated R2R code is as good as tier 1 code, or if we were to use NGen for CoreLib, those would all map to `Optimized` instead of mapping to `ReadyToRun` or `NGen`, as they would not be tiered up. Not sure if it would be worth distinguishing those, thoughts?